spec/features/collection_drafts/forms/spatial_information_form_spec.rb

```diff
@@ -122,6 +122,12 @@
         expect(page).to have_content('Collection Draft Updated Successfully!')
       end
 
+      it 'contains the correct dropdown options' do
+        within '#draft_tiling_identification_systems_0_tiling_identification_system_name' do
+          page.has_select?('Tiling Identification System Name', with_options: ['CALIPSO', 'MISR', 'MODIS Tile EASE', 'MODIS Tile SIN', 'WELD Alaska Tile', 'WELD CONUS Tile', 'WRS-1', 'WRS-2', 'Military Grid Reference System'])
```

Comment:
This should use `options:` and not `with_options:`
with_options is checking partial set match, meaning adding new options doesn't cause the test to fail.  I think we want this test to be brittle.